### PR TITLE
Reagent 1.1.0

### PR DESCRIPTION
**Pull request by Deraen**

### CODE REVIEW

**Comment by yogthos** [comment]
👍

File: resources/leiningen/new/reagent/project.clj [hunk 1]
```diff
@@ -6,14 +6,17 @@
...
                  [reagent-utils "0.3.3"]
+                 [cljsjs/react "17.0.2-0"]
+                 [cljsjs/react-dom "17.0.2-0"]
                  [ring "1.8.1"]
                  [ring/ring-defaults "0.3.2"]
                  [hiccup "1.0.5"]
                  [yogthos/config "1.1.7"]
                  [org.clojure/clojurescript "1.10.866"
                   :scope "provided"]
+                 [com.google.javascript/closure-compiler-unshaded "v20210302"]
```
Comment (yogthos) [inline resources/leiningen/new/reagent/project.clj:19]: should this be marked as provided, it should only be needed for compiling the js artifact so probably shouldn't end up in the jar for the backend?

Comment (Deraen) [reply resources/leiningen/new/reagent/project.clj:19]: Oops. That one shouldn't be there in fact. I added the correct closure-compiler for shadow-cljs to a separate profile. When not using shadow-cljs, the version clojurescript depends on can be used.

Comment (yogthos) [reply resources/leiningen/new/reagent/project.clj:19]: ah ok, I'll just remove it then :)

**Pull request merged**